CHANGELOG.md

```diff
@@ -34,6 +38,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   with Secrets Provider [cyberark/sidecar-injector#76](https://github.com/cyberark/sidecar-injector/pull/76)
 
 ### Security
+- Updated replace statements to force golang.org/x/net v0.0.0-20220923203811-8be639271d50
+   and updated testify to 1.8.0 [cyberark/sidecar-injector#77](https://github.com/cyberark/sidecar-injector/pull/78)
```

Comment:
should this be [cyberark/sidecar-injector#78]

Reply:
Yep. should be. Missed one place. Thanks!